racket/collects/racket/vector.rkt

```diff
@@ -94,7 +94,9 @@
 ;; uses name for error reporting
 (define (one-arg-check f v name)
   (unless (and (procedure? f) (procedure-arity-includes? f 1))
-    (raise-argument-error name "(any/c . -> . any/c)" 0 f)))
+    (raise-argument-error name "(any/c . -> . any/c)" 0 f))
```

Comment:
If I read correctly, `0` is provided here as a `bad-pos`, so we are using the second form of [`raise-argument-error`](https://docs.racket-lang.org/reference/exns.html#%28def._%28%28quote._~23~25kernel%29._raise-argument-error%29%29). As a result, there should be one more argument at the end: `v`.